src/main/java/com/cinema/dao/OrderDao.java

```diff
@@ -6,4 +6,6 @@
 
 public interface OrderDao extends AbstractDao<Order> {
     List<Order> getOrdersHistory(User user);
+
+    List<Order> getOrdersHistory(Long userId);
```

Comment:
let's remove this method and use `List<Order> getOrdersHistory(User user);` that we already have

Reply:
done